feature/eks-cluster-creation/vars-secret.tf-example

```diff
@@ -0,0 +1,7 @@
+// Secrets - do not included in repo
```

Comment:
Using ENVS would be most desirable approach I think from automation point of view. What do you think?

Reply:
I am not an expert here, but I'm not sure, if its worth, as tis is very simple solution and works - we need  just replace 2 fields as in var file. If you decide that tfvars will be better solution, I can update the code..

Reply:
Personaly I'm using profiles it's just `export AWS_PROFILE=your_profile_name` and thanks to that I don't have to paste my credentials somewhere.
But this file is in gitignore so there is no risk to be commited to repo. So this red warning will not happened by default: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#static-credentials
The biggest question for me is if this will work with env variables. If so approved for me, if not we have to decide as a team which way we want to use.

Reply:
Yeah, those are good comments. @mum0l please create new issue regarding those concerns, but lets finish this task asap.

Reply:
I just test it, and looks like when we remove the `access_key` and `secret_key` from provider "aws" section, than using export command for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY we're able to deploy it without setting the vars-secret.tf file ;)